.github/ISSUE_TEMPLATE/release.md

```diff
@@ -1,59 +1,60 @@
 ---
-name: k6 Release 
-about: k6 release accommodates activities and a checklist that has the k6 open-source release process.
+name: k6 Release
+about: k6 release accommodates activities and a checklist with the k6 open-source release process.
 title: 'k6 release v0.4x.0'
+labels: ["release"]
 ---
 
-**Release Date**: 
+**Release Date**:
 
 25th May 2023 <<  TODO: WRITE HERE THE UPDATED RELEASE DATE
 
 ## Release Activities
 
 ### At the beginning of the cycle
 
-- [ ] Create a new branch `release-notes-v04x` and add a new related [release notes file](release-template/release%20notes) using the available [template](release-template/release%20notes/template.md).
-- [ ] Go through the potential [dependencies updates](Dependencies.md) then create a PR if any is relevant. 
+- [ ] Create a new `release-notes-v04x` branch and add a new release notes file using the available [template](release notes/template.md) to the [repository's `release notes` folder](/release notes).
+- [ ] Go through the potential [dependencies updates](Dependencies.md) and create a dedicated PR if any of them is relevant to this release.
 
 ### Release Preparation
 
-~ 1 week before the release date.
+#### ~ 1 week before the release date.
 
-- [ ] k6-docs PRs for all new or changed functionality have been created.
-- [ ] All PRs to the k6 repository in the current [milestone](https://github.com/grafana/k6/milestones) have been merged.
-- [ ] A pull request with the release notes and request the final reviews (including the @k6-browser, devrels folks).
-- [ ] Open a separate PR for bumping [the version](https://github.com/grafana/k6/blob/9fa50b2d1f259cdccff5cc7bc18a236d31c345ac/lib/consts/consts.go#L11).
-- [ ] The release notes PR shared in the `#k6-oss-dev` internal channel mentioning all the impacted teams (@k6-browser, @k6-chaos, @k6 devrel and any other potential stackholder of the new release).
-- [ ] `DefinitelyTyped/DefinitelyTyped` PR(s) is ready.
+- [ ] Ensure all PRs in the k6-docs repository, related to new or modified functionalities introduced by the new version have been merged.
+- [ ] Ensure all PRs in the k6 repository, part of the current [milestone](https://github.com/grafana/k6/milestones), have been merged.
+- [ ] Open a PR with the release notes for the new version, and ask teams who might have contributed to the release (@k6-browser, @k6-chaos, @devrel teams, etc.) to contribute their notes and review the existing ones.
+- [ ] Share the release notes PR in the `#k6-oss-dev` internal slack channel, mentioning and asking for contributions of all the impacted teams (@k6-browser, @k6-chaos, @k6 devrel, and any other potential stakeholder of the new release).
```

Comment:
Maybe we shouldn't add such internal things. 

Idially those also won't be needed by anything else but pinging people

Reply:
You're right, it felt strange putting it there, going to rephase it 👍🏻